terminus-core/src/components/appRoot.component.pug

```diff
@@ -3,99 +3,133 @@ title-bar(
   [class.inset]='hostApp.platform == Platform.macOS'
 )
 
-.content(
-    [class.tabs-on-top]='config.store.appearance.tabsLocation == "top"'
-)
-    .tab-bar
-        .inset.background(*ngIf='hostApp.platform == Platform.macOS && config.store.appearance.frame == "thin" && config.store.appearance.tabsLocation == "top"')
-        .tabs(
-            dnd-sortable-container,
-            [sortableData]='app.tabs',
-        )
-            tab-header(
-                *ngFor='let tab of app.tabs; let idx = index',
-                dnd-sortable,
-                [sortableIndex]='idx',
-                (onDragStart)='onTabDragStart()',
-                (onDragEnd)='onTabDragEnd()',
+.wrap
+  .tab-bar.vertical(
```

Comment:
I wanted to make tab-bar one component, but it was difficult and I couldn't do it well :confounded: